### PR TITLE
Add Madara Source: Reader's Point

### PR DESCRIPTION
**Pull request by Shota5749**
Checklist:

- [ ] Updated `extVersionCode` value in `build.gradle` for individual extensions
- [ ] Updated `overrideVersionCode` or `baseVersionCode` as needed for all multisrc extensions
- [ ] Referenced all related issues in the PR body (e.g. "Closes #xyz")
- [ ] Added the `isNsfw = true` flag in `build.gradle` when appropriate
- [ ] Have not changed source names
- [ ] Have explicitly kept the `id` if a source's name or language were changed
- [ ] Have tested the modifications by compiling and running the extension through Android Studio

### CODE REVIEW

File: multisrc/src/main/java/eu/kanade/tachiyomi/multisrc/madara/MadaraGenerator.kt [hunk 1]
```diff
@@ -348,6 +348,7 @@ class MadaraGenerator : ThemeSourceGenerator {
         SingleLang("RawDEX", "https://rawdex.net", "ko", isNsfw = true, overrideVersionCode = 1),
         SingleLang("ReadAdult", "https://readadult.net", "en", isNsfw = true, overrideVersionCode = 1),
         SingleLang("ReadManhua", "https://readmanhua.net", "en", overrideVersionCode = 2),
+        SingleLang("Reader's Point", "https://readers-point.space/", "en"),
```
[the remark this answers is not in the document]
Comment (Shota5749) [reply multisrc/src/main/java/eu/kanade/tachiyomi/multisrc/madara/MadaraGenerator.kt]: Added

File: multisrc/src/main/java/eu/kanade/tachiyomi/multisrc/madara/MadaraGenerator.kt [hunk 2]
```diff
@@ -348,7 +348,7 @@ class MadaraGenerator : ThemeSourceGenerator {
         SingleLang("RawDEX", "https://rawdex.net", "ko", isNsfw = true, overrideVersionCode = 1),
         SingleLang("ReadAdult", "https://readadult.net", "en", isNsfw = true, overrideVersionCode = 1),
         SingleLang("ReadManhua", "https://readmanhua.net", "en", overrideVersionCode = 2),
-        SingleLang("Reader's Point", "https://readers-point.space/", "en"),
+        SingleLang("Reader's Point", "https://readers-point.space", "en", className = "Reader's Point"),
```
Comment (jopejoe1) [inline multisrc/src/main/java/eu/kanade/tachiyomi/multisrc/madara/MadaraGenerator.kt:351]: Remove the space and `'` in class name as it only supports letters & digits

**Review by jopejoe1 — CHANGES_REQUESTED** [review]

File: multisrc/src/main/java/eu/kanade/tachiyomi/multisrc/madara/MadaraGenerator.kt (hunk 2 above)
Comment (Shota5749) [reply multisrc/src/main/java/eu/kanade/tachiyomi/multisrc/madara/MadaraGenerator.kt]: Edited

**Comment by CarlosEsco** [comment]
Still has build issues

**Comment by alessandrojean** [comment]
Would be better to use [this tool] to generate the icons to keep the consistency with the others.

About the build errors, I don't see anything wrong in the code, maybe it's just some random failure (as already happened before).

A build rerun could help.

[this tool]: https://as280093.github.io/AndroidAssetStudio/icons-launcher.html#foreground.type=clipart&foreground.clipart=android&foreground.space.trim=1&foreground.space.pad=0.25&foreColor=rgba(96%2C%20125%2C%20139%2C%200)&backColor=rgb(68%2C%20138%2C%20255)&crop=0&backgroundShape=square&effects=none&name=ic_launcher

**Comment by jopejoe1** [comment]
Removing the `'` from the source name should fix the build errors.

**Comment by Riztard** [comment]
> Removing the `'` from the source name should fix the build errors.

on their website their name didnt use `'` too 
or maybe need `\`

**Pull request closed without merging**